bib/cmd/bootc-image-builder/main.go

```diff
@@ -101,6 +101,23 @@ func loadConfig(path string) (*BuildConfig, error) {
 	return &conf, nil
 }
 
+func getContainerSize(imgref string) (uint64, error) {
```

Comment:
Very minor nit but I have tried in code I've been writing recently to add a docstring even for functions like this.  This is maybe a good example because when talking about sizes the units are important, and while it'd be surprising if the units were anything other than bytes...probably still worth stating?

(One thing I don't understand is why Go has the convention of repeating the type name, it makes quick doc comments more annoying to write)

```suggestion
// getContainerSize returns the size of an already pulled container image in bytes
func getContainerSize(imgref string) (uint64, error) {
```

Reply:
Excellent suggestion, added